assets/src/stories-editor/helpers/index.js

```diff
@@ -1774,3 +1774,17 @@ export const ensureAllowedBlocksOnPaste = ( blocks, clientId, isFirstPage ) => {
 	} );
 	return allowedBlocks;
 };
+
+/**
+ * Given a block client ID, returns the corresponding DOM node for the block,
+ * if exists. As much as possible, this helper should be avoided, and used only
+ * in cases where isolated behaviors need remote access to a block node.
+ *
+ * @param {string} clientId Block client ID.
+ * @param {Element} scope an optional DOM Element to which the selector should be scoped
+ *
+ * @return {Element} Block DOM node.
+ */
+export const getBlockDOMNode = ( clientId, scope = document ) => {
+	return scope.querySelector( '[data-block="' + clientId + '"]' );
```

Comment:
Could use template literals here

Reply:
```suggestion
	return scope.querySelector( `[data-block="${ clientId }"]` );
```

Reply:
This was copied and pasted. 
https://github.com/WordPress/gutenberg/blob/68dc4f7f5708d30fe18e070241b4a52401d1b251/packages/block-editor/src/utils/dom.js#L12